drivers/remote/driver.go

```diff
@@ -13,28 +13,11 @@ var errNoCallback = errors.New("No Callback handler registered with Driver")
 const remoteNetworkType = "remote"
 
 type driver struct {
-	networkType string
-	callback    driverapi.DriverCallback
```

Comment:
You seem to be moving the goalposts somewhat. The code I'm replacing doesn't do any integration. Yes, it provides a `registerRemoteDriver` method, but it just invokes the DriverCallback, with no suggestion of how it could be used.

To address your concern, though: the intention is that Init() hooks into the plugin system, as in your example from the other day. I can't provide that code here, however, since it would rely on plugin pkg code being released in some official form (I don't know what the bar for official is).